### PR TITLE
Pupils at expected standards at the end of key stage 2

### PR DESCRIPTION
**Pull request by bailed22**

### CODE REVIEW

File: src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv [hunk 1]
```diff
@@ -0,0 +1,1162 @@
...
+E09000033,Westminster,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2016-09-01T00:00:00/P1Y,68.0,Percentage,%,,,,1,
+E09000033,Westminster,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2021-09-01T00:00:00/P1Y,65.0,Percentage,%,,,,1,
+E09000033,Westminster,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2015-09-01T00:00:00/P1Y,58.0,Percentage,%,,,,1,
+E13000002,"Outer London
```
Comment (GregoryJPavier) [inline src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv:754]: Row has been separated across two lines, as have several rows below.

File: src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv [hunk 2]
```diff
@@ -0,0 +1,1162 @@
...
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2020-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2021-09-01T00:00:00/P1Y,63.0,Percentage,%,,,,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving publ
```
Comment (GregoryJPavier) [inline src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv:901]: Row has been separated across two lines, as have several rows below.

**Review by GregoryJPavier — COMMENTED** [review]
The CSV needs a little tidying up, but that's it.

File: src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv (hunk 1 above)
Comment (bailed22) [reply src/store/csv/stub/pupils-at-expected-standards-by-end-of-primary-school.csv]: done

File: src/store/csv/stub/content/pupils-at-expected-standards/2023-03/1.csv [hunk 3]
```diff
@@ -0,0 +1,1155 @@
...
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2020-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2021-09-01T00:00:00/P1Y,63.0,Percentage,%,,,,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2015-09-01T00:00
```
Comment (GregoryJPavier) [inline src/store/csv/stub/content/pupils-at-expected-standards/2023-03/1.csv:893]: These rows need to be fixed, several rows are spread out across two lines.

Comment (bailed22) [reply src/store/csv/stub/content/pupils-at-expected-standards/2023-03/1.csv:893]: completed

File: src/store/csv/stub/content/pupils-at-expected-standards/2023-03/.~lock.1.csv# [hunk 4]
```diff
@@ -0,0 +1 @@
+,davidbailey,Davids-MBP.broadband,17.01.2024 13:29,file:///Users/davidbailey/Library/Application%20Support/LibreOffice/4;
```
Comment (GregoryJPavier) [inline src/store/csv/stub/content/pupils-at-expected-standards/2023-03/.~lock.1.csv#:1]: I assume this is auto-generated or something, but it needs to be removed.

Comment (bailed22) [reply src/store/csv/stub/content/pupils-at-expected-standards/2023-03/.~lock.1.csv#:1]: okay

**Review by GregoryJPavier — APPROVED** [review]

**Pull request merged**